packages/files/file.pony

```diff
@@ -1,4 +1,21 @@
-primitive _FileHandle
+use @_read[I32](fd: I32, buffer: Pointer[None], bytes_to_read: I32) if windows
+use @read[ISize](fd: I32, buffer: Pointer[None], bytes_to_read: USize) if not windows
+use @_write[I32](fd: I32, buffer: Pointer[None], bytes_to_send: I32) if windows
+use @writev[ISize](fd: I32, buffer: Pointer[None], num_to_send: I32) if not windows
+use @_lseeki64[I64](fd: I32, offset: I64, base: I32) if windows
+use @lseek[I64](fd: I32, offset: I64, base: I32) if not windows
```

Comment:
That setting changes the way `ponyc` and `libponyrt` are compiled. It doesn't change the way a Pony program is compiled. So a change in `threads.c` will indeed pick the right `lseek`, as would any C program you might check when that `#define` is set. It may be that `libponyrt` then forces the function alias in Pony programs, but that's what I would like to double check.